task02/index.js

```diff
@@ -0,0 +1,81 @@
+
+const FIGURE = {
+    RECTANGLE:  {value: 1, name: 'Прямоугольник', code: 'R'},
+    TRIANGLE:   {value: 2, name: 'Треугольник', code: 'T'},
+    CIRCLE:     {value: 3, name: 'Окружность', code: 'C'}
+}
+
+class Rectangle {
+    constructor(height, width){
+        this.height = height;
+        this.width = width;
+    }
+
+    Area() {
+        return this.height * this.width;
+    }
+}
+class Triangle {
+    constructor(height, base) {
+        this.height = height;
+        this.base = base;
+    }
+
+    Area() {
```

Comment:
```suggestion
    area() {
```
метод с маленькой прописываем:)

Reply:
Я привык в C# так делать - все методы и свойства класса с большой буквы